BinaryContract.sol

```diff
@@ -1,3 +1,62 @@
 pragma solidity >=0.4.22 <0.7.0;
 
-//TODO
\ No newline at end of file
+//TODO
+contract BinaryContract{
+
+    struct Transaction {
+        address from;
+        address to;
+        uint amount;
+        uint date;
+    } // this is only needed if we're keeping a transaction log inside a bin contract as well
+
+    struct ContractDebt{
+        address debtor;
+        uint amountOwned;
+    }
+
+
+    address playerOne;
+    address playerTwo;    
+    ContractDebt currentDebt;
+
+    Transaction[] binContractTransactionsLog; 
+    /* not sure if needed because the transactions are already saved in the 
+    group contract and a two person group i.e binary contract is just a two person group contract with a log already being made.
+    My question is: do we need to differentiate between a binary contract and a two person group? */ 
+
+    constructor() public{
+        //TODO
+    }
+
+    function updateContractDebt public (address memory inserter, uint memory amount) {
```

Comment:
I am not sure that I understood that function's role, let's talk about it in a meeting.
When we write code (and there is not actually any other code) it is harder to understand, I guess.